jruby-gradle-base-plugin/src/main/groovy/com/github/jrubygradle/JRubyExec.groovy

```diff
@@ -199,7 +210,11 @@ class JRubyExec extends JavaExec implements JRubyExecTraits {
      */
     @Override
     List<String> getArgs() {
-        JRubyExecUtils.buildArgs(jrubyArgs, getScript(), scriptArgs)
+        List<String> extra = []
+        if (JRUBYEXEC_CONFIG != jrubyConfigurationName) {
```

Comment:
These needs some comments on why it is here for posterity (read: so I don't accidentally delete it in the future :P)